src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsGroupComponent.scala

```diff
@@ -14,7 +16,19 @@ trait RawlsGroupComponent {
     def * = (groupName, groupEmail) <> (RawlsGroupRecord.tupled, RawlsGroupRecord.unapply)
   }
 
-  val rawlsGroupQuery = TableQuery[RawlsGroupTable]
+  object rawlsGroupQuery extends TableQuery(new RawlsGroupTable(_)) {
+    def save(group: RawlsGroup): ReadWriteAction[RawlsGroup] = {
```

Comment:
had to implement this to get the test data off the ground as groups are required by workspaces